.mergify.yml

```diff
@@ -5,7 +5,8 @@ pull_request_rules:
       - "#approved-reviews-by>=1"
       - label=ready-to-merge
       - label!=hold-off-merging
-      - status-success=build
+      - check-success=Build app
+      - check-success=Lint Helm chart
```

Comment:
Can you try to validate the config on https://dashboard.mergify.com/github/infracloudio/repo/botkube/config-editor ?

Reply:
Works well 👍 
![Screenshot 2022-05-31 at 13 49 07](https://user-images.githubusercontent.com/7155799/171166298-57e44a6d-fc21-43b3-bf15-3052aa7527b3.png)